DependencyInjection/DoctrineExtension.php

```diff
@@ -892,6 +906,32 @@ protected function loadCacheDriver($cacheName, $objectManagerName, array $cacheD
                 ));
         }
 
+        if ($isPsr6 === null) {
+            $definition = $container->getDefinition($serviceId);
+            $isPsr6     = is_a($definition->getClass(), CacheItemPoolInterface::class, true);
+        }
+
+        $cacheName = str_replace('_cache', '', $cacheName);
+
+        // Create a wrapper as required
+        if ($isPsr6 && ! $usePsr6) {
+            $wrappedServiceId = sprintf('cache.doctrine.orm.provider.%s.%s', $objectManagerName, $cacheName);
```

Comment:
Why not reusing `createPoolCacheDefinition` function? It's now unused. Either use it or remove it.

Reply:
I missed that. Will give that another pass.

Reply:
I think this comment still applies. Also, are you sure with this naming? On most of the places we have doctrine.orm prefix

Reply:
The unused `createPoolCacheDefinition` was removed and the cache name was changed for consistency.